ThirdAI/Terms and Conditions/third_AI.py

```diff
@@ -0,0 +1,34 @@
+from thirdai import licensing, neural_db as ndb
```

Comment:
Use docstring and comments, to make the program easily understandable to others as well as you.